### DISCUSSION TITLE
How to implement a real time chart whose touch gesture is updated in tooltip and toched spot even when the mouse cursor is not moving

### DISCUSSION THREAD

**Discussion by universal-101** [imaNNeo/fl_chart#1415] [General]
Hi, As the title suggests, I've a real time data coming from socket, which I want to plot with tool tips. I'm first trying my hand at the fl_chart example project, specifically line chart sample 10, as it is closest to what I want to implement. 
![initial gif](https://github.com/imaNNeo/fl_chart/assets/132750600/aecb241e-42e9-4c5b-a873-5099806619af)
![running gif](https://github.com/imaNNeo/fl_chart/assets/132750600/8145dc78-63ee-4e02-aab1-c6af18b99237)

Currently, the behavior is as shown here in gif. But, how to make the tool tip and touched spot follow the cursor position when the data is being added. Basically, the touched spot and tool tip only updates when i move the mouse cursor and do not update their position when the data is updated in the chart.
Thanks a lot

**Comment by imaNNeo** [comment]
It is a bug actually, when the chart's data is changed, we need to respond to the touch event again.
Can you please search the issues and create one if you cannot find any related issues about this bug?
